cmd/kuberhealthy/config.go

```diff
@@ -13,15 +13,17 @@ import (
 // Config holds all configurable options
 type Config struct {
 	kubeConfigFile            string
-	ListenAddress             string `yaml:"listenAddress,omitempty"`
-	EnableForceMaster         bool   `yaml:"enableForceMaster,omitempty"`
-	LogLevel                  string `yaml:"logLevel,omitempty"`
-	InfluxUsername            string `yaml:"influxUsername,omitempty"`
-	InfluxPassword            string `yaml:"influxPassword,omitempty"`
-	InfluxURL                 string `yaml:"influxURL,omitempty"`
-	InfluxDB                  string `yaml:"influxDB,omitempty"`
-	EnableInflux              bool   `yaml:"enableInflux,omitempty"`
-	ExternalCheckReportingURL string `yaml:"externalCheckReportingURL,omitempty"`
+	ListenAddress             string        `yaml:"listenAddress,omitempty"`
+	EnableForceMaster         bool          `yaml:"enableForceMaster,omitempty"`
+	LogLevel                  string        `yaml:"logLevel,omitempty"`
+	InfluxUsername            string        `yaml:"influxUsername,omitempty"`
+	InfluxPassword            string        `yaml:"influxPassword,omitempty"`
+	InfluxURL                 string        `yaml:"influxURL,omitempty"`
+	InfluxDB                  string        `yaml:"influxDB,omitempty"`
+	EnableInflux              bool          `yaml:"enableInflux,omitempty"`
+	ExternalCheckReportingURL string        `yaml:"externalCheckReportingURL,omitempty"`
+	CheckCleanupDuration      time.Duration `yaml:"checkCleanupDuration,omitempty"`
```

Comment:
rename to `JobCleanupDuration` since this only applies to khjobs cleanup not khchecks
```suggestion
	JobCleanupDuration      time.Duration `yaml:"jobCleanupDuration,omitempty"`
```